File: voicepy/voicepy/rogivoice.py
import rclpy
from rclpy.node import Node
from std_msgs.msg import String

class voicepy:
    def __init__(self,node_handle):
        self.voice_pub = node_handle.create_publisher(String, 'voice', 10)
    def say(self, text):
        msg = String()
        msg.data = text
        self.voice_pub.publish(msg)
        

File: voicecpp/src/rogivoice.cpp
#include "rogivoice.hpp"
#include "rclcpp/rclcpp.hpp"
#include "std_msgs/msg/string.hpp"

RogiVoice::RogiVoice(rclcpp::Node *node):node(node){
  voice_pub = node->create_publisher<std_msgs::msg::String>("voice", 10);
}

void RogiVoice::say(std::string voice_name){
  auto msg = std_msgs::msg::String();
  msg.data = voice_name;
  voice_pub->publish(msg);
}